### PR TITLE
[docs] batch 5 updating functions

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

Add examples to the sql-functions.md page

Updates the DIV, HUMAN_READABLE_BINARY_BYTE_FORMAT, HUMAN_READABLE_DECIMAL_BYTE_FORMAT, HUMAN_READABLE_DECIMAL_FORMAT, and SAFE_DIVIDE numeric functions.

This PR has:

- [X] been self-reviewed.

### CODE REVIEW

File: docs/querying/sql-functions.md [hunk 1]
```diff
@@ -1430,11 +1503,28 @@ Trims characters from the trailing end of an expression.
...
+* **Syntax:** `SAFE_DIVIDE(x, y)`
+* **Function type:** Scalar, numeric 
 
-Returns `x` divided by `y`, guarded on division by 0.
+<details><summary>Example</summary>
+
+The following calculates divisions of integer `78` by integer `10`.
+
+```sql
+  SELECT SAFE_DIVIDE(78, 10) AS "safe_division"
```
Comment (asdf2014) [inline docs/querying/sql-functions.md:1516]:
```suggestion
SELECT SAFE_DIVIDE(78, 10) AS "safe_division"
```

File: docs/querying/sql-functions.md [hunk 2]
```diff
@@ -605,11 +605,34 @@ Returns the rank for a row within a window without gaps. For example, if two row
...
+  SELECT DIV(78, 10) as "division"
+  ``` 
+
+  Returns the following:
+
+  | `division` |
+  | -- |
+  | `7` |
+
+</details>
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:627]: Comment out the docs (`<!-- -->`) for this function. It is false until Druid 31. We can fix it closer to the release.

Comment (techdocsmith) [reply docs/querying/sql-functions.md:627]: Keep the admonition.

File: docs/querying/sql-functions.md [hunk 3]
```diff
@@ -866,28 +889,78 @@ Returns a union of HLL sketches.
...
-`HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])`
+Converts an integer byte size into human-readable [IEC](https://en.wikipedia.org/wiki/Binary_prefix) format.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])`
+* **Function type:** Scalar, numeric
+
+<details><summary>Example</summary>
+
+  Converts `1000000` into IEC format.
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:899]:
```suggestion
  The following converts `1000000` into IEC format.
```
Retain "the following for consistency"

**Review by techdocsmith — CHANGES_REQUESTED** [review]
Good progress. Request that we not include docs for the div function until it is available.

File: docs/querying/sql-functions.md (hunk 2 above)
Comment (edgar2020) [reply docs/querying/sql-functions.md:627]: commented out the <Details> box and all its content

File: docs/querying/sql-functions.md (hunk 3 above)
Comment (edgar2020) [reply docs/querying/sql-functions.md]: Changed it to "The following example..."

Comment (edgar2020) [reply docs/querying/sql-functions.md]: as well as in similar instances

**Review by techdocsmith — APPROVED** [review]
LGTM

**Pull request merged**